storm-client/test/jvm/org/apache/storm/assignments/LocalAssignmentsBackendTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.assignments;
+
+import org.apache.commons.collections.map.HashedMap;
+import org.apache.storm.generated.Assignment;
+import org.apache.storm.generated.NodeInfo;
+import org.apache.storm.utils.ConfigUtils;
+import org.junit.Test;
+
+import java.util.*;
+
+public class LocalAssignmentsBackendTest {
+
+    @Test
+    public void testLocalAssignment(){
+        Map<String, Assignment> stormToAssignment = new HashMap<>();
+        String storm1 = "storm1";
+        String storm2 = "storm2";
+        Assignment ass1 = mockedAssignment(1);
+        Assignment ass2 = mockedAssignment(2);
+
+        ILocalAssignmentsBackend backend = LocalAssignmentsBackendFactory.getBackend(ConfigUtils.readStormConfig());
+        assert (null == backend.getAssignment(storm1));
```

Comment:
Let's use JUnit Assert class instead of using Java assert in unit test. assertEquals, assertTrue, assertFalse, etc.

Reply:
okey